packages/runtime/src/widgets/Flex.tsx

```diff
@@ -0,0 +1,91 @@
+import { useMemo } from "react";
+import {
+  CustomScope,
+  CustomScopeProvider,
+  useCustomScope,
+  useRegisterBindings,
+  useTemplateData,
+  type EnsembleWidget,
+} from "@ensembleui/react-framework";
+import { isEmpty, isObject, omit } from "lodash-es";
+import { WidgetRegistry } from "../registry";
+import { EnsembleRuntime } from "../runtime";
+import {
+  type EnsembleWidgetStyles,
+  type EnsembleWidgetProps,
+  type HasItemTemplate,
+} from "../shared/types";
+
+interface FlexStyles extends EnsembleWidgetStyles {
+  direction?: "horizontal" | "vertical" | undefined;
+  gap?: string;
+  lineGap?: string;
+  overflow?: string;
+}
+
+export type FlexProps = {
+  children?: EnsembleWidget[];
+} & HasItemTemplate &
+  EnsembleWidgetProps<FlexStyles>;
+
+export const FlexWidget: React.FC<FlexProps> = (props) => {
+  const { "item-template": itemTemplate, children, ...rest } = props;
```

Comment:
Why not just render a Row/Column based on the direction property?

Reply:
so you are suggesting instead of creating new widget, we can introduce the direction property on the row and column layout

Reply:
No I mean keep this widget, just return the different component based on direction.

Reply:
Something like, is direction is column then return the column widget, otherwise return the row widget ?